### ISSUE TITLE
Map multiple ports to several name

### ISSUE THREAD

**Issue by no-more** [vfarcic/docker-flow-proxy#59]
Hi,

I'm using this tool with manual configuration, and I have a service running which expose several ports (currently 2 but later 3).
When I'm adding proxy definition with different target name for example 
 - myService:80 -> myservice.test.com
 - myService:8080 -> mybackendservice.test.com

but currently the last definition override the previous one.
Is there any way to handle this in the current version, or maybe would it be possible to make this evolve ?

Thanks a lot for this great tool.

**Comment by no-more** [comment]
I have tried to understand how the code is working (never used GO), according to what I've understood, an update is required to handle my case.
This update would requires to add port to the name in the configuration file.

Do you thing you could add this feature ?
This would greatly help me (and I think other people too)

Thanks.

**Comment by vfarcic** [comment]
I can add an option to specify different ACLs for a single service. I'll take a look at the code during the new weekend and let you know when I'll be able to implement it.

**Comment by no-more** [comment]
Hi,

Thanks a lot, that's a great news.

**Comment by vfarcic** [comment]
Sorry for not responding earlier. I had too much work on my plate and could not find a free moment to dedicate to this issue.

Can you confirm one thing? Would it be acceptable if you simply use multiple domains? For example:

```sh
curl "$PROXY_IP:8080/v1/docker-flow-proxy/reconfigure?serviceName=go-demo&servicePath=/&serviceDomain=myservice.test.com,mybackendservice.test.com"
```

That would allow you to send requests to:

myService:80 -> myservice.test.com
myService:80 -> mybackendservice.test.com
myService:8080 -> myservice.test.com
myService:8080 -> mybackendservice.test.com

The only problem is that this approach would not explicitly deny access to some combinations. Please let me know whether that would work. It would be a very small change to the code.

**Comment by vfarcic** [comment]
The feature is done and passed automated tests. There's only one task left before I release it. I need to run a few manual tests with a live service. The problem is that I don't have any service that listens to multiple ports. All those that I have are a microservice/single-process/single-port type of services.

Do you have some service that I could use for testing this feature? That would save me from writing a demo app just for this issue.

**Comment by no-more** [comment]
Hi,

Thanks a lot but I don't have any public repro that could be used for this. Maybe if you build a test release I could check on my side and tell you the results.

Thanks a lot.

**Comment by vfarcic** [comment]
The feature is finished and available from the [release 1.333](https://github.com/vfarcic/docker-flow-proxy/releases/tag/1.133). Please reopen this issue if you have any problems with it or if something is missing.
